.functions.php

```diff
@@ -35,3 +29,51 @@ function checkIfAdmin() {
     return ($results[0]['admin'] == 1 || $results[0]['schedco'] == 1);
   }
 }
+
+function getUser($sessionID, $connection){
+  if(isset($sessionID)){
+    $connection->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+  
+    if(!isset($dname)) {
+      $dname = 'ambulanc_web';
+    }
+  
+    $connection->exec("USE `$dname`");
+  
+    $statement = $connection->prepare('SELECT * FROM members LEFT JOIN sessions on sessions.userID = members.id WHERE sessions.sessionID = :sessionID');
+    $statement->bindParam(":sessionID", $sessionID);
+    $statement->execute();
+    $results=$statement->fetchAll(PDO::FETCH_ASSOC);
+    updateSession($sessionID, $connection);
+    return json_encode($results[0]);
```

Comment:
This function should just return the result set as is and then for the one time you echo this out to the browser, then you can do a json_encode there.

Reply:
Fixed